code/datums/craft/step.dm

```diff
@@ -231,8 +231,7 @@
 		return I
 
 	//The item is on the user
-	if (user.canUnEquip(I))
```

Comment:
Why switch this from a mob function to a generic one? What's the gain here, how does this help?

Reply:
Mob proc checks only mob stats but generic one checks both `mob.canUnequip()` and `item.canBeUnequipped()`

Reply:
No. I replaced one function with one complex which made of two.
`item.MobCanUnequip(...)` with `mob_can_unequip(Item, Mob,...)` which calls `Mob.canUnequip(Item, ...)` and `Item.canBeUnequipped(Mob,..)`